keel-bakery-plugin/src/main/kotlin/com/netflix/spinnaker/keel/bakery/DefaultBaseImageCache.kt

```diff
@@ -10,7 +10,7 @@ class DefaultBaseImageCache(
     baseImages[os]?.get(label.name.toLowerCase()) ?: throw UnknownBaseImage(os, label)
 }
 
-@ConfigurationProperties("keel.plugins.bakery.base-images")
+@ConfigurationProperties(prefix = "keel.plugins.bakery")
```

Comment:
There was one path too many in the prefix here, since Spring uses the name of the variable(s) in the class to determine the leaf property name in the config file.